Vienna/Resources/de.lproj/Predicates.strings

```diff
@@ -1,12 +1,8 @@
-
 "%[All,Any,None]@ of the following are true" = "Artikel erfüllt %1$[alle,beliebige,keine]@ der folgenden Bedingungen";
-
 "%[Deleted,Flagged,HasEnclosure,Read]@ is %[No,Yes]@" = "%1$[Ist gelöscht,Ist markiert,Hat Anhang,Ist gelesen]@ %2$[Nein,Ja]@";
-
-"%[Author,Folder,Subject,Text]@ %[is,is not]@ %@" = "%1$[Autor,Ordner,Betreff,Inhalt]@ %2$[ist,ist nicht]@ %3$@";
-
+"%[Author,Folder,Subject,Text]@ %[is,is not]@ %@" = "%1$[Autor,Feed oder Ordner,Betreff,Inhalt]@ %2$[ist,ist nicht]@ %3$@";
 "%[Author,Subject,Text]@ %[contains,does not contain]@ %@" = "%1$[Autor,Betreff,Inhalt]@ %2$[enthält,enthält nicht]@ %3$@";
-
-"%[Date]@ %[is,is greater than,is greater than or equal to,is less than,is less than or equal to]@ %[last week,today,yesterday]@" = "%1$[Datum]@ %2$[ist,ist nach,ist nach oder ist,ist vor,ist vor oder ist]@ %3$[letzte Woche,heute,gestern]@";
-
-"%[Date]@ %[less than ago,more than ago]@ %[minutes,hours,days,months,weeks,years]@ %@" = "%1$[Datum]@ %2$[vor weniger als,vor mehr als]@ %4$@ %3$[Minute(n),Stunde(n),Tag(en),Monat(en),Woche(n),Jahr(en)]@";
+"%[Date]@ %[is,is greater than,is greater than or equal to,is less than,is less than or equal to]@ %[last week]@" = "%1$[Datum]@ %2$[ist,ist nach,ist nach oder ist,ist vor,ist vor oder ist]@ %3$[letzte Woche]@";
+"%[Date]@ %[is,is greater than or equal to,is less than,is less than or equal to]@ %[yesterday]@" = "%1$[Datum]@ %2$[ist,ist nach oder ist,ist vor,ist vor oder ist]@ %3$[gestern]@";
+"%[Date]@ %[is,is less than,is less than or equal to]@ %[today]@" = "%1$[Datum]@ %2$[ist,ist vor,ist vor oder ist]@ %3$[heute]@";
+"%[Date]@ %[less than ago,more than ago]@ %[days,hours,minutes,months,weeks,years]@ %@" = "%1$[Datum]@ %2$[vor weniger als,vor mehr als]@ %4$@ %3$[Tag(en),Stunde(n),Minute(n),Monat(en),Woche(n),Jahr(en)]@";
```

Comment:
Here we must write "vor höchstens,vor mehr als" instead of "vor weniger als,vor mehr als".